test/Container/ApplicationFactoryTest.php

```diff
@@ -65,6 +65,60 @@ public function getRouterFromApplication(Application $app)
         return $r->getValue($app);
     }
 
+    public function testFactorySetsPhpSettingsFromConfig()
+    {
+        $this->container
+            ->has('Zend\Expressive\Router\RouterInterface')
+            ->willReturn(false);
+        $this->container
```

Comment:
I agree that it adds noise, but for some reason I couldn't bypass those opening lines within the `ApplicationFactory::__invoke()`. I'll revisit that anyhow.